webapp/views/App/views/Data/MapView/CoordinateAttributeDataLayer/CoordinateAttributeMarker.js

```diff
@@ -37,33 +43,48 @@ export const CoordinateAttributeMarker = (props) => {
   const { showMarkersLabels } = options
 
   return (
-    <CircleMarker
-      center={[latitude, longitude]}
-      radius={markerRadius}
-      color={markersColor}
-      fillColor={markersColor}
-      fillOpacity={fillOpacity}
-    >
-      {showMarkersLabels && (
-        <Tooltip
-          ref={tooltipRef}
-          direction="top"
-          offset={[0, -10]}
-          opacity={tooltipOpacity}
-          permanent
-          onOpen={onTooltipOpen}
-        >
-          {ancestorsKeys.join(' - ')}
-        </Tooltip>
-      )}
-      <CoordinateAttributePopUp
-        attributeDef={attributeDef}
-        point={point}
-        recordUuid={recordUuid}
-        parentUuid={parentUuid}
-        ancestorsKeys={ancestorsKeys}
-        onRecordEditClick={onRecordEditClick}
-      />
-    </CircleMarker>
+    <div>
+      <CoordinateAttributePolygon latitude={latitude} longitude={longitude} />
+      <CircleMarker
+        center={[latitude, longitude]}
+        radius={markerRadius}
+        color={markersColor}
+        fillColor={markersColor}
+        fillOpacity={fillOpacity}
+      >
+        {showMarkersLabels && (
+          <Tooltip
+            ref={tooltipRef}
+            direction="top"
+            offset={[0, -10]}
+            opacity={tooltipOpacity}
+            permanent
+            onOpen={onTooltipOpen}
+          >
+            {ancestorsKeys.join(' - ')}
+          </Tooltip>
+        )}
+        <CoordinateAttributePopUp
+          attributeDef={attributeDef}
+          point={point}
+          recordUuid={recordUuid}
+          parentUuid={parentUuid}
+          ancestorsKeys={ancestorsKeys}
+          onRecordEditClick={onRecordEditClick}
+        />
+      </CircleMarker>
+    </div>
   )
 }
+
+CoordinateAttributeMarker.propTypes = {
+  ancestorsKeys: PropTypes.any,
```

Comment:
the type should be PropTypes.array, so you avoid to define propTypes for ancestorKeys too